docs/content/connectors/mysql-cdc.md

```diff
@@ -361,6 +361,14 @@ After the server you monitored fails in MySQL cluster, you only need to change t
 
 It's recommended to configure a DNS(Domain Name Service) or VIP(Virtual IP Address) for your MySQL cluster, using the DNS or VIP address for ```mysql-cdc``` connector, the DNS or VIP would automatically route the network request to the active MySQL server. In this way, you don't need to modify the address and restart your pipeline anymore.
 
+#### MySQL Heartbeat Event Support
```

Comment:
Could we introduce an option `heartbeat.interval.ms` with default value `30s` so that  user can configure this easily?